egs2/TEMPLATE/asr2/asr2.sh

```diff
@@ -680,6 +680,12 @@ fi
 if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ] && ! [[ " ${skip_stages} " =~ [[:space:]]4[[:space:]] ]]; then
     log "Stage 4a: Perform Kmeans using ${kmeans_feature_type} features"
 
+    if [ ${ngpu} -gt 0 ]; then
```

Comment:
Sorry, I don't want to mix `ngpu` (usually used in training LM / ASR models) with `use_gpu` here.

Reply:
Added `gpu_kmeans` parameter (following the `gpu_inference` style variable naming), and set it into default in https://github.com/espnet/espnet/pull/5441/commits/91c2b06069b95492b9f6eee9ffb5e1cc84bdb896 Thanks for pointing out!